### PR TITLE
Can do crypto patch

### PR DESCRIPTION
**Pull request by Kakemone**

### CODE REVIEW

File: protobuf/can_do_crypto.proto [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+package can_do_crypto;
+
+import "psa_key_attributes.proto";
+
+enum CheckType {
+  Use = 0;
+  Generate = 1;
+  Import = 2;
+  Derive = 3;
```
Comment (ionut-arm) [inline protobuf/can_do_crypto.proto:15]: Enum names should be all-caps, as per [the style guide](https://developers.google.com/protocol-buffers/docs/style#enums)

File: protobuf/can_do_crypto.proto [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+ * SPDX-License-Identifier: Apache-2.0
+ */
+syntax = "proto3";
+
+package can_do_crypto;
+
+import "psa_key_attributes.proto";
+
+enum CheckType {
+  Use = 0;
```
Comment (ionut-arm) [inline protobuf/can_do_crypto.proto:12]: I think we usually leave aside the variant with value 0 to be some "default" value (in other enums we seem to have called them `<Enum name>_NONE`). This avoids the possibility of any value being selected non-explicitly, just by virtue of the field missing in the message - clients must send a specific code, explicitly

**Review by ionut-arm — COMMENTED** [review]
Thanks for the new contract!

**Review by ionut-arm — APPROVED** [review]
Thanks for the changes 👌🏻

**Review by hug-dev — APPROVED** [review]

**Pull request merged**